docs/book/psr3.md

```diff
@@ -61,3 +61,6 @@ $logger->addProcessor(new Zend\Log\Processor\PsrPlaceholder);
 $logger->info('User with email {email} registered', ['email' => 'user@example.org']);
 // logs message 'User with email user@example.org registered'
 ```
+## Usage with ServiceManager
+
+For usage with ServiceManager, check this [PsrLoggerAbstractServiceFactory](./service-manager.md#PsrLoggerAbstractServiceFactory).
```

Comment:
* Use the full name for the service manager here: "zend-servicemanager" to make clear what is meant.
* All class names must be set in backticks: \`PsrLoggerAbstractServiceFactory\`
* Instead of "check this" we should use a formulation like "see the Service Manager Integration section" and set the link on "Service Manager Integration section".
* For linking to page in the same the directory it is enough to use the filename without any dots or slashes. (see [MkDocs documentation](https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages))